subo/command/dev.go

```diff
@@ -33,8 +33,19 @@ func DevCmd() *cobra.Command {
 			}
 
 			port, _ := cmd.Flags().GetString("port")
+			verbose, _ := cmd.Flags().GetBool("verbose")
 
-			_, err = util.Run(fmt.Sprintf("docker run -v=%s:/home/atmo -e=ATMO_HTTP_PORT=%s -p=%s:%s suborbital/atmo:%s atmo", bctx.Cwd, port, port, port, bctx.AtmoVersion))
+			envvar := ""
+
+			if verbose == true {
+				envvar = "-e ATMO_LOG_LEVEL=debug"
+			}
+
+			//dockerCmd stores the Docker command to be displayed and executed
+			dockerCmd := fmt.Sprintf("docker run -v=%s:/home/atmo -e=ATMO_HTTP_PORT=%s %s -p=%s:%s suborbital/atmo:%s atmo", bctx.Cwd, port, envvar, port, port, bctx.AtmoVersion)
+			fmt.Printf("\nRunning Docker command :\n%s\n\n", dockerCmd)
+
+			_, err = util.Run(fmt.Sprintf("docker run -v=%s:/home/atmo -e=ATMO_HTTP_PORT=%s %s -p=%s:%s suborbital/atmo:%s atmo", bctx.Cwd, port, envvar, port, port, bctx.AtmoVersion))
```

Comment:
because you already have the parsed docker command in the `dockerCmd` variable, you don't need to do the `fmt.Sprintf` here again
```suggestion
			_, err = util.Run(dockerCmd)
```

Reply:
Right!